omero/sysadmins/unix/server-centos6-ius.txt

```diff
@@ -0,0 +1,217 @@
+Installing Python 2.7 and Ice 3.5 on CentOS 6
+=============================================
+
+Running OMERO on CentOS 6 has a number of special requirements which
+deviate from the standard installation instructions.  The instructions
+below will set up Python 2.7 and Ice 3.5 on CentOS 6. We tested the
+installation with Python 2.7 from `IUS <https://ius.io/>`_ and used
+a `virtual environment <https://virtualenv.readthedocs.org/en/latest/>`_. 
+
+Python 2.7
+----------
+
+CentOS 6 provides Python 2.6.  However, OMERO.web requires Python 2.7
+in order to use django 1.8.  While django 1.6 may be used with Python
+2.6, this version of django no longer has security support.  In
+consequence, it is necessary to upgrade to Python 2.7 in order to
+obtain django security updates, which are required for a production
+deployment.
+
+As root, run::
+
+    wget https://centos6.iuscommunity.org/ius-release.rpm
+    rpm -Uvh ius-release*.rpm
+    yum install python27
+
+To use this version of Python, it is preferrable to install
+the various dependencies in a virtual environment. 
+If virtualenv is not already installed, install it using pip run::
+
+	yum install python27-pip
+	pip2.7 install virtualenv
+
+To demonstrate its use::
+
+    $ python --version
+    Python 2.6.6
+    $ virtualenv /home/omero/omeroenv
+    $ source /home/omero/omeroenv/bin/activate
+    $ python --version
+    Python 2.7.10
+    $ deactivate # back to the default python
+
+Development packages
+--------------------
+
+The following sections require a number of libraries and a compiler.
+Run the following commands to install them::
+
+    yum groupinstall "Development Tools"
+    yum install epel-release
+    # Ice dependencies
+    yum install expat-devel bzip2-devel openssl-devel zlib-devel
+    # Python module dependencies
+    yum install hdf5-devel freetype-devel libjpeg-devel libpng-devel
+
+
+Ice 3.5
+-------
+
+The RPM packages provided by ZeroC for CentOS 6 use the Python 2.6
+provided by the system.  OMERO will need an Ice build which uses
+Python 2.7. A version of Ice 3.5 built using Python 2.7 installed
+above is available at http://downloads.openmicroscopy.org/ice/experimental.
+
+The following steps will install Ice into :file:`/opt`::
+
+    # Download and install needed Ice build dependencies (db and mcpp)
+    curl -o /etc/yum.repos.d/zeroc-ice-el6.repo http://download.zeroc.com/Ice/3.5/el6/zeroc-ice-el6.repo
+    mkdir /home/ice-download
+    cd /home/ice-download
+    wget http://downloads.openmicroscopy.org/ice/experimental/Ice-3.5.1-b1-centos6-iuspy27-x86_64.tar.gz
+    yum install db53 db53-devel db53-utils mcpp-devel 
```

Comment:
`*-devel` packages shoudn't be needed if you're using the precompiled Ice package.